css/colors.css

```diff
@@ -0,0 +1,23 @@
+:root{
+    --primary-color:rgb(196,85,168);
+    --secondary-color:rgb(222,205,245);
+    --complement-color:rgb(69,207,233);
+    --green-color: rgb(124,252,0);
+    --red-color:rgb(255, 0, 0);
+    --yellow-color:#ffcc00;
```

Comment:
Better to be consistent in using either HEX values or rgb values to colors.

Reply:
thank you for pointing it out changed it to rgb value.